code/game/atoms.dm

```diff
@@ -2082,14 +2082,14 @@
 //Update the screentip to reflect what we're hoverin over
 /atom/MouseEntered(location, control, params)
 	. = ..()
-	// Statusbar
-	status_bar_set_text(usr, name)
 	// Screentips
-	if(usr?.hud_used)
-		if(!usr.hud_used.screentips_enabled || (flags_1 & NO_SCREENTIPS_1))
-			usr.hud_used.screentip_text.maptext = ""
+	var/datum/hud/active_hud = usr.hud_used
+	if(active_hud)
+		if(!active_hud.screentips_enabled || (flags_1 & NO_SCREENTIPS_1))
+			active_hud.screentip_text.maptext = ""
 		else
-			usr.hud_used.screentip_text.maptext = MAPTEXT("<span style='text-align: center'><span style='font-size: 32px'><span style='color:[usr.hud_used.screentip_color]: 32px'>[name]</span>")
+			//We inline a MAPTEXT() here, because there's no good way to statically add to a string like this
+			active_hud.screentip_text.maptext = "<span class='maptext'><span style='text-align: center'><span style='font-size: 32px'><span style='color:[active_hud.screentip_color]: 32px'>[name]</span></span>"
```

Comment:
```suggestion
			active_hud.screentip_text.maptext = "<span class='maptext' style='text-align: center; font-size: 32px; color: [active_hud.screentip_color]'>[name]</span>"
```

this has been bugging me forever. theres even a stupid `:32px` at the end that just does nothing